app.py

```diff
@@ -64,6 +63,24 @@
     f.write(css)
 
 
+class RunOptimizationReturn(NamedTuple):
+    solution_map: str
+    cost_table: tuple
+    hybrid_table_label: str
+    sampler_type: str
+    reset_results: bool
+    parameter_hash: str
+    performance_improvement_quantum: str
+    cost_comparison: dict
+    problem_size: int
+    search_space: str
+    wall_clock_time_classical: str
+    wall_clock_time_quantum: str
+    num_locations: int
+    vehicles_deployed: int
+    results_tab: str
```

Comment:
I'd recommend probably moving this into the `run_optimization` function and type hint the return in quotes (`...  -> "RunOptimizationReturn":`), alt. put it right above the function using it.

Reply:
Turns out dash uses pickle, and pickle doesn't like pickling objects from classes defined inside the local scopes of functions. 🥒🫙🫤